go/consensus/tendermint/abci/state.go

```diff
@@ -278,6 +278,11 @@ func (s *applicationState) OwnTxSigner() signature.PublicKey {
 	return s.ownTxSigner
 }
 
+func (s *applicationState) currentEpoch(ctx *Context) (epochtime.EpochTime, error) {
```

Comment:
Looks like a good idea to have this exposed (also see #2625 which adds this too under a different name).